src/Commands/FormatCommandCommon.cs

```diff
@@ -27,44 +26,80 @@ internal static class FormatCommandCommon
-        internal static readonly Option<string> VerbosityOption = new Option<string>(new[] { "--verbosity", "-v" }, Resources.Set_the_verbosity_level_Allowed_values_are_quiet_minimal_normal_detailed_and_diagnostic).AcceptOnlyFromAmong(VerbosityLevels);
-        internal static readonly Option BinarylogOption = new Option<string>(new[] { "--binarylog" }, Resources.Log_all_project_or_solution_load_information_to_a_binary_log_file)
+        internal static readonly CliOption<string[]> ExcludeDiagnosticsOption = new("--exclude-diagnostics")
         {
-            ArgumentHelpName = "binary-log-path",
-            Arity = ArgumentArity.ZeroOrOne
-        }.AcceptLegalFilePathsOnly();
-        internal static readonly Option ReportOption = new Option<string>(new[] { "--report" }, Resources.Accepts_a_file_path_which_if_provided_will_produce_a_json_report_in_the_given_directory)
+            AllowMultipleArgumentsPerToken = true,
+            DefaultValueFactory = _ => Array.Empty<string>(),
+            Description = Resources.A_space_separated_list_of_diagnostic_ids_to_ignore_when_fixing_code_style_or_3rd_party_issues,
+        };
+        internal static readonly CliOption<string> SeverityOption = new CliOption<string>("--severity")
         {
-            ArgumentHelpName = "report-path",
-            Arity = ArgumentArity.ZeroOrOne
-        }.AcceptLegalFilePathsOnly();
+            Description = Resources.The_severity_of_diagnostics_to_fix_Allowed_values_are_info_warn_and_error,
+        };
+        internal static readonly CliOption<string[]> IncludeOption = new("--include")
+        {
+            AllowMultipleArgumentsPerToken = true,
+            DefaultValueFactory = _ => Array.Empty<string>(),
+            Description = Resources.A_list_of_relative_file_or_folder_paths_to_include_in_formatting_All_files_are_formatted_if_empty,
+        };
+        internal static readonly CliOption<string[]> ExcludeOption = new("--exclude")
+        {
+            AllowMultipleArgumentsPerToken = true,
+            DefaultValueFactory = _ => Array.Empty<string>(),
+            Description = Resources.A_list_of_relative_file_or_folder_paths_to_exclude_from_formatting,
+        };
+        internal static readonly CliOption<bool> IncludeGeneratedOption = new("--include-generated")
+        {
+            Description = Resources.Format_files_generated_by_the_SDK,
+        };
+        internal static readonly CliOption<string> VerbosityOption = new CliOption<string>("--verbosity", "-v")
+        {
+            Description = Resources.Set_the_verbosity_level_Allowed_values_are_quiet_minimal_normal_detailed_and_diagnostic,
+        };
+        internal static readonly CliOption<string> BinarylogOption = new CliOption<string>("--binarylog")
+        {
+            HelpName = "binary-log-path",
+            Arity = ArgumentArity.ZeroOrOne,
```

Comment:
In general, you shouldn't need to set the arity unless there's some custom parsing going on. Otherwise, it's inferred from the generic type of the `Option<T>` or `Argument<T>`.